app/views.py

```diff
@@ -90,61 +94,88 @@ def progressreport():
     """Renders the report page."""
     return render_template('report.html', title=' | Progress Report')
 
-#@app.route('/createclass', methods=['GET', 'POST'])
-#def createclass():
-#    """Renders the create class page for educators."""
-#    form = CreateName()
-#    if form.validate_on_submit():
-#        temp = set_code(6)
-#        while Group.query.filter_by(classCode=temp).first() is not None:
-#            temp = set_code(6)
-#        group = Group(name=form.name.data, classCode=temp)
-#        db.session.add(group)
-#        db.session.commit()
-#        return redirect(url_for('createclasssuccess'))
-#    return render_template('createclass.html', title=' | Create Class', form=form)
-
-@app.route('/createclasssuccess', methods=['GET'])
-def createclasssuccess():
+@app.route('/class', methods=['POST'])
+def createclass():
+    """Renders the create class page for educators."""
+    if not current_user.check_educator():
+        return render_template('error404.html'), 404
+    classForm = CreateName(prefix='class')
+    quizForm = CreateName(prefix='quiz')
+    image_file = url_for('static', filename='resources/images/profile_pics/' + current_user.image_file)
```

Comment:
Why are you adding the profile image for? The class has no images?

Reply:
This was yours. It's the user's own profile image.